EXT_SpeciesType.md

```diff
@@ -0,0 +1,90 @@
+Convention for Specifying Particle Species 
+==========================================
+
+openPMD extension name: `SpeciesType`
+
+openPMD extension ID: `4`
+
+
+Introduction
+------------
+
+This convention is for standardizing the names of particle species, e.g. in
+particle physics.
+
+
+Additional Record Attribute
+---------------------------
+
+The following additional attribute for openPMD `records` is defined in this
+extension:
+
+- `SpeciesType`
+  - type: *(string)*
+  - scope: *optional*
+  - description: particle species in this record. If there are multiple
+                 species to be specified, they can be specified using a
+                 semicolon separated list.
+  - allowed values:
+    - *see the lists below* and additionally
+    - `other` if none of the ones below applies, user are free to append a
+      free text after a colon, e.g. `other:neutralino` or `other:cherry`
+  - examples:
+    - `electron` (e.g. on an electron `particle record` or an electron
+                  density `mesh record`)
+    - `electron;proton;#12C` (e.g. on a `mesh record` for a plasma's
+                              local charge density)
+    - `other:apple;other:orange` (for a `record` mixing apples & oranges)
+
+This attribute can be used with any `record` (including `mesh records`).
+
+### SubAtomic Particles
+
+  - `subatomic:antimuon`
```

Comment:
offline discussion: we won't collide (yet) and if we should later one we can still add prefixes and convert known ones.